tests/Mcfedr/AwsPushBundle/Tests/Controller/ApiControllerTest.php

```diff
@@ -15,12 +15,12 @@ public function testRegisterDevice()
         $client = self::createClient();
 
         $snsMock = $this->getMockBuilder('Aws\Sns\SnsClient')
-            ->setMethods(['createPlatformEndpoint'])
+            ->addMethods(['createPlatformEndpoint'])
             ->disableOriginalConstructor()
             ->getMock();
         $snsMock->expects($this->once())
             ->method('createPlatformEndpoint')
-            ->willReturn('some:arn');
+            ->willReturn(['EndpointArn' => 'some:arn']);
```

Comment:
not sure we want to change the api

Reply:
@mcfedr, We don't change the API, we just fix the invalid mock response which didn't failed on the older versions of PHPUnit:
```
1) Mcfedr\AwsPushBundle\Tests\Controller\ApiControllerTest::testRegisterDevice
TypeError: Cannot access offset of type string on string

src/Mcfedr/AwsPushBundle/Service/Devices.php:73
src/Mcfedr/AwsPushBundle/Controller/ApiController.php:68
vendor/symfony/http-kernel/HttpKernel.php:152
vendor/symfony/http-kernel/HttpKernel.php:74
vendor/symfony/http-kernel/Kernel.php:202
vendor/symfony/http-kernel/HttpKernelBrowser.php:65
vendor/symfony/framework-bundle/KernelBrowser.php:168
vendor/symfony/browser-kit/AbstractBrowser.php:370
tests/Mcfedr/AwsPushBundle/Tests/Controller/ApiControllerTest.php:29
```

Method `registerDevice` calls `createPlatformEndpoint`  
https://github.com/mcfedr/awspushbundle/blob/17990f036cb072f34b6fffbb1cdc93cbecd84fd5/src/Mcfedr/AwsPushBundle/Service/Devices.php#L60
and returns 
https://github.com/mcfedr/awspushbundle/blob/17990f036cb072f34b6fffbb1cdc93cbecd84fd5/src/Mcfedr/AwsPushBundle/Service/Devices.php#L79
therefore the response of `createPlatformEndpoint` must be an array with the single element `EndpointArn`.
Documentation of the `createPlatformEndpoint` states the same: https://docs.aws.amazon.com/sns/latest/api/API_CreatePlatformEndpoint.html#API_CreatePlatformEndpoint_ResponseElements
So there is no change in API, but only fix the test.